e2e/scripts/natsserver.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+# for our tests we need to port-forward the Pods.
+kubectl -n kyma-system port-forward eventing-nats-0 8222:8222 &
+PID1=$!
+kubectl -n kyma-system port-forward eventing-nats-1 8223:8222 &
+PID2=$!
+kubectl -n kyma-system port-forward eventing-nats-2 8224:8222 &
+PID3=$!
```

Comment:
This assumes that there are 3 NATS instances. Is this always the case?

Reply:
yes, the test setup has always 3 nats pods, here